### ISSUE TITLE
LG 2-7: Isn't Event Storming already from 2013?

### ISSUE THREAD

**Issue by mattes3** [isaqb-org/curriculum-ddd#13]
In the text it says EventStorming (Brandolini, 2021). This should be 2013, instead.

**Comment by hofstef** [comment]
Yes, the method was first published in 2013. But (Brandolini, 2021) is a reference to the book. The newest version on Leanpub is from 2021. Hence, this issue is a "won't fix".
